submissions/akinbobolaFaith/phpMySqlDatabase/customersList.php

```diff
@@ -0,0 +1,41 @@
+<?php
+require_once("cookie.php");
+
+if (isset($_COOKIE["customers"])) {
+    $data = json_decode($_COOKIE["customers"], true);
+    } else {
+    return $data;
```

Comment:
No, this doesn't work that way. Kindly place your 
![image](https://user-images.githubusercontent.com/43061141/105823223-2b02e600-5fbd-11eb-93a7-30914f8b59ae.png) in a function. So that function return `$data` as you can see already. SO, if cookie is not set, you should call or invoke that function instead. The function will then run the query again, store it is a cookie and run $data.